misago/users/tests/test_anonymizeoldips.py

```diff
@@ -1,43 +1,45 @@
 import datetime
-import pytz
 
 from django.contrib.auth import get_user_model
 from django.core.management import call_command
-from django.test import TestCase
-from django.utils.six import StringIO
 from django.utils import timezone
+from django.utils.six import StringIO
+from django.test import TestCase, override_settings
```

Comment:
Is there a reason why you have moved the `from django.test` line? It was placed corretly before this change.